### PR TITLE
ISSUE-147: Update docs to match PR146

### LINKED ISSUE #147: Update docs to match PR146
#What?

@DiegoPino has done an incredible amount of work to prepare for release, and now the docs need to be updated for info and with formatting both as local repo docs and for the official documentation pages.

### PR DESCRIPTION
**Pull request by aksm**
Resolves #147

### CODE REVIEW

File: docs/ubuntu.md [hunk 1]
```diff
@@ -78,13 +80,14 @@ Ok, now we are ready to start.
 
 ## Step 1: Deployment
 
-##### Prefer to watch a video of how to install? Go to our [`user contributed documentation`](https://github.com/esmero/archipelago-deployment/blob/1.0.0-RC1/docs/ubuntu.md#user-contributed-documentation)!
+##### Prefer to watch a video of how to install? Go to our [`user contributed documentation`](#user-contributed-documentation-a-video)!
```
Comment (DiegoPino) [inline docs/ubuntu.md]: @aksm this video might not be updated enough anymore.. @alliomeria what do you think?

Comment (alliomeria) [reply docs/ubuntu.md]: @DiegoPino & @aksm think some might still find referencing this older video to be helpful context. Maybe update the description to something like "You may find this 'user contributed tutorial video' which was created for an earlier Archipelago release to be helpful. Please note there are significant differences between the executed steps, and that you need to follow the current release instructions in order to have a successful deployment." Too much? Better to remove for now & add a reference back later on for an updated video?

Comment (DiegoPino) [reply docs/ubuntu.md]: Ok, that sounds good. But be sure its not a "1:1" doc for this release.@aksm can you make the change?
We also need the M1 docker-compose, will submit in 10/15 minutes, need to fetch lunch or something and be back

Comment (aksm) [reply docs/ubuntu.md]: @DiegoPino @alliomeria ok, added a footnote. Let me know if you think that works or doesn't.

**Review by DiegoPino — APPROVED** [review]
Ok, let's merge. I will update M1 info once we test with @alliomeria

**Pull request merged**